content/02.introduction.md

```diff
@@ -119,9 +120,68 @@ Pathogenesis is made up of three major components: entry, replication, and sprea
+
+Once a pathogen has entered the body, the immune system needs to recognize it.
+The first line of defense fronted by the innate immune response allows the body to respond to pathogen-associated molecular patterns (PAMPs), which are molecular signatures associated with the presence of foreign particles that are likely to be pathogenic.
+For example, double-stranded RNA (dsRNA) is not produced endogenously, and thus its presence suggests that viral replication is likely to be occurring.
+By responding to this signature, the immune response is able to target viruses it has never encountered before just by recognizing them as viruses.
+Through this response, the innate immune system allows cells to collect information about the pathogen to prime the adaptive immune response, which can target specific protein sequences associated with individual pathogens (e.g., a specific viral species or even strain).
+Several types of leukocytes (white blood cells) are professional phagocytic cells that can detect and engulf a particle, enveloping it in a phagosome.
+The conditions within the phagosome are then modulated to destroy the pathogen through fusion of the phagosome with a lysosome or granule to form a phagolysosome.
+Monocytes, which include both dendritic cells and macrophages, are types of phagocytic white blood cells that are also professional antigen-presenting cells, which meaning that they can then present the resulting viral peptides on the cell surface via Major Histocompatability Complex II (MHC-II) proteins.
+This allows the cell to break the pathogen into its component peptides, which allow the body to organize a response to the specific pathogen.
+
+The adaptive immune response is the second line of defense once a pathogen has entered the body.
+It, in turn, can be broadly classified into the humoral and cell-mediated responses. 
+The humoral response includes soluble proteins and antibodies that can neutralize and eliminate pathogens in circulation that have not yet entered cells, and it can also mediate the killing of pathogen-infected cells. 
+The cell-mediated response includes the killing of pathogen-infected cells by cytotoxic CD8+ T cells and the secretion of cytokines to drive inflammation and support immune cell activities.
+The immune response therefore describes the complex interplay between pathogens infecting hosts and hosts responding with a multifaceted immune defense; a pathogen's virulence depends on its success in evading the immune system or otherwise manipulating the host.
+Innovation in therapeutics often seeks to supplement or bolster the immune response in order to enhance the body's ability to eliminate a pathogen.
+However, disease symptoms can be caused either by a pathogen or by the immune response to the pathogen, and a host immune system's efforts to fight an infection can also sometimes cause significant damage to the host [@doi:978-0815332183; @doi:9780071283663].
```

Comment:
Are both of these DOI citations actually ISBN citations?

Reply:
@agitter they are both standard textbooks, if that's what you're asking?

Reply:
```suggestion
However, disease symptoms can be caused either by a pathogen or by the immune response to the pathogen, and a host immune system's efforts to fight an infection can also sometimes cause significant damage to the host [@isbn:9780815332183; @isbn:9780071283663].
```

Thanks for confirming @yemarshall.  I wanted to check whether we could change the Manubot citations as suggested here.   These Manubot citations give:
```
Molecular biology of the cell
Bruce Alberts (editor)
Garland Science (2002)
ISBN: 9780815332183

Vander’s human physiology: the mechanisms of body function
Eric P Widmaier, Hershel Raff, Kevin T Strang
McGraw-Hill Higher Education (2008)
ISBN: 9780071283663
```

Reply:
that looks right. (The first textbook has multiple authors but the editor shown above is the first author so that works.)